### DISCUSSION TITLE
`<leader>/` is not dot repeatable?

### DISCUSSION THREAD

**Discussion by vypxl** [NvChad/NvChad#1717] [Q&A]
Hi, back again with another question :)

So, I like the binding `<leader>/` to toggle comments. However, this shortcut, unlike the builtin `gcc` of [numToTStr/Comment.nim](/numToStr/Comment.nvim) is not dot-repeatable. Instead, if I press `.` after using it, nvim
repeats the command before that.

I investigated why this is and located the issue to [core/mappings.lua](https://github.com/NvChad/NvChad/blob/main/lua/core/mappings.lua#L118), where nvchad manually calls `Comment.nvim` instead of the `<Plug>(comment_toggle_linewise_current)` it provides.

Now, just changing the mappings to use that does not work, because packer has to load the plugin first before you can map this. Packer is instructed by nvchad to load the plugin when the user presses `gc` or `gb`. So if I press `gc`, then use the updated `<leader>/` mapping later, it works as expected. It does not when I first press `<leader>/` after opening nvim. Then it is permanently broken for the session.

So, how do I resolve this? I could just eagerly load `Comment.nvim`, but that seems crude. I cannot just add `<leader>/` to the load trigger keys packer takes, because then the mapping is broken (Why that happens is explained [here](https://github.com/wbthomason/packer.nvim/issues/1078)). I also tried mapping `<leader>/` to `gcc`, but that does not trigger packer to load the plugin.

I also feel like this should be working by default in nvchad if it's possible to implement elegantly.

What to do?

**Comment by siduck** [comment]
i dont quit get the issue, `<leader>/` just toggles comment, does it not work there?

**Comment by vypxl** [comment]
My problem is that I press `<leader>/`, then move one line down and press `.` to do it again, but it does not toggle comment, instead repeating whatever I did before commenting.
